src/rest-server/src/models/token.js

```diff
@@ -28,21 +27,27 @@ const { encodeName } = require('@pai/models/v2/utils/name');
 // job-specific tokens and other tokens are saved in different namespaces
 const userTokenNamespace =
   process.env.PAI_USER_TOKEN_NAMESPACE || 'pai-user-token';
-const jobTokenNamespace =
-  process.env.PAI_JOB_TOKEN_NAMESPACE || 'pai-job-token';
 
 // create namespace if not exists
 if (process.env.NODE_ENV !== 'test') {
   k8sModel.createNamespace(userTokenNamespace);
-  k8sModel.createNamespace(jobTokenNamespace);
 }
 
-const sign = async (username, application, expiration) => {
+const sign = async (
+  username,
+  application,
+  expiration,
+  jobSpecific = 'false',
```

Comment:
while use string `'false'` instead of bool `false` here?

Reply:
> while use string `'false'` instead of bool `false` here?

Typo, fixed